tests/testing_constants.py

```diff
@@ -6,8 +6,8 @@
 TOKEN = "hf_94wBhPGp6KrrTH3KDchhKpRxZwd6dmHWLL"
 
 ENDPOINT_PRODUCTION = "https://huggingface.co"
-ENDPOINT_STAGING = "https://moon-staging.huggingface.co"
-ENDPOINT_STAGING_BASIC_AUTH = f"https://{USER}:{PASS}@moon-staging.huggingface.co"
+ENDPOINT_STAGING = "https://hub-ci.huggingface.co"
+ENDPOINT_STAGING_BASIC_AUTH = f"https://{USER}:{PASS}@hub-ci.huggingface.co"
```

Comment:
BTW is the new CI env already seeding that user/org?

Reply:
Yes !

Reply:
yes !